### PR TITLE
README edits

### PR DESCRIPTION
**Pull request by ismangil**
Editing of README text for misspellings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,13 +78,13 @@ sh get-docker.sh
...
 > You can read more about this at [`docker-install`](https://github.com/docker/docker-install)
 
-To compile the project, assuming that your already cloned this repository, just
+To compile the project, assuming that you already cloned this repository, just
 execute the `build` target, this will generate in `build` folder all the binaries
 and debian packages:
 
 ```sh
 > make build
-> ls -1 build/
```
Comment (zen) [inline README.md]: It's not necessary mistake ;) Often used in scripts to feed only filenames

**Comment by ismangil** [comment]
@zen I assumed it meant `ls -l` to list the built package name in full

**Comment by zen** [comment]
@ismangil I think it would be really hard to mix "1" with "l", so I thought is intentional. It's not critical, but we can wait for @mcuadros

File: README.md (hunk 1 above)
Comment (mcuadros) [reply README.md]: Yep, it's intentional, I wanted `-1`

**Comment by ismangil** [comment]
Done, back to `ls -1`

**Pull request merged**